website/docs/pendant/pendant.md

```diff
@@ -0,0 +1,53 @@
+---
+id: pendant
+title: Pendant Overview
+---
+![pendant 3](https://user-images.githubusercontent.com/64480560/210899367-c57a8bae-c450-4f71-bc68-5f05d337c896.jpg)
+
+### Description
+The pendant is a wired Manual joystick control interface with a rubberized backlit keypad that is  waterproof and rugged. This CANbus enabled controller is pluggable anywhere on the tractor communication backbone.
+
+### How to connect
+
+The pendant must be connect to CAN/Power accessory port which is location on the back of the dashboard panel. (please reference photo below)
+
+![187535995-d4ad8c7c-05a0-482b-8be9-6b6fb6b87ac3](https://user-images.githubusercontent.com/64480560/210901594-6733b85d-292b-4468-9c65-3e275ed1c6f5.png)
+
+
+### Pendant Controls
+
+![IMG-9902](https://user-images.githubusercontent.com/64480560/211636387-fa11f5ea-59b0-4506-8005-d99ab21f90c6.jpg)
+
+**Joystick** (omni directional stick that is used to control the amiga. This controls include, Acceleration, and turning left and right, reverse)
+
+**Cruise(X)** – this will allow you to place your Amiga into cruise control. Press this button while the amiga is completely stopped. To activate this, press `cruise` then gently push forward the joystick to accelerate, pull back on joystick to decrease acceleration
+
+:::caution
+Pushing forward and pulling back on the joystick does not control speed, only acceleration and how much it will accelerate in relation to the set speed on the dashboard home tab.
+:::
+
+**Brake(O)**
+this is an emergency brake for the MCUs, pressing this will end cruise control and stop all motors
```

Comment:
MCUs? What this means should be defined, but also I think we use MCU elsewhere to mean microcontroller (as with MCU kit)